modules/custom/openy_search/src/Config/OpenySearchOverrides.php

```diff
@@ -50,7 +50,6 @@ public function loadOverrides($names) {
       if (in_array($default_theme . '.settings', $names)) {
         $overrides[$default_theme . '.settings'] = [
           'search_query_key' => 'q',
-          'search_page_alias' => 'search',
```

Comment:
We should not completely remove the overriding of `search_page_alias` config. We should replace hardcoded `search` alias with correct alias from the search results landing node.